monitor.cpp

```diff
@@ -18,37 +55,70 @@ void displayWarning() {
 /// @param pulseRate
 /// @param spo2
 /// @return 1 if vitals are ok, 0 otherwise
-int vitalsOk(float temperature, float pulseRate, float spo2) {
-  if (checkVitals(temperature, pulseRate, spo2)) {
-    return 1;
-  }
-  return 0;
+bool isVitalsOk(float temperature, float pulseRate, float spo2) {
+  initializeWarnigMessages();
+  return (isTemperatureOk(temperature) && isPulseRateOk(pulseRate) && isSpo2Ok(spo2));
+}
+
+bool isItInRange(float value, float lower, float upper) {
+     return (value >= lower && value <= upper);
 }
 
-bool checkVitals(float temperature, float pulseRate, float spo2) {
-  return (checkTemperature(temperature) && checkPulseRate(pulseRate) && checkSpo2(spo2));
+void displayMessage(int MsgInd, Language lang) {
+  // Set console output to UTF-8
+  // SetConsoleOutputCP(CP_UTF8);
+    system("chcp 65001");
+    if (MsgInd > 4) {
+        MsgInd = 4;
+    }
+    // Print the wide string
+    std::wcout.imbue(std::locale(std::locale(), new std::codecvt_utf8<wchar_t>));
```

Comment:
Good approach... I always have difficulty printing utf-8 on the console. Learnt something today from you :)
Appreciate the effort to find this method - to convert utf-8 to wchar